File: BMangMall/app/src/main/java/com/banshion/bmangmall/MainActivity.java
package com.banshion.bmangmall;

import androidx.appcompat.app.AppCompatActivity;

import android.graphics.Typeface;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

import com.banshion.bmangmall.utils.ToastUtil;

public class MainActivity extends AppCompatActivity implements View.OnClickListener{

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Button loginBtn = findViewById(R.id.login_btn);
        loginBtn.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        switch (v.getId()) {
            case R.id.login_btn:
                EditText userName = findViewById(R.id.username_et);
                EditText password = findViewById(R.id.password_et);
                if ("admin".equals(userName.getText().toString()) && "admin".equals(password.getText().toString())) {
                    ToastUtil.showMsg(MainActivity.this, "登录成功", Toast.LENGTH_SHORT);
                } else {
                    ToastUtil.showMsg(MainActivity.this, "登录失败", Toast.LENGTH_SHORT);
                }
                break;
            default:
                break;
        }
    }
}

File: UIWidgetTestKotlin/app/src/main/java/com/banshion/uiwidgettestkotlin/MainActivity.kt
package com.banshion.uiwidgettestkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Toast
import androidx.appcompat.app.AlertDialog
import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.title.*

class MainActivity : AppCompatActivity(), View.OnClickListener {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        supportActionBar?.hide()
//        button.setOnClickListener(this)
    }

    override fun onClick(v: View?) {
        when (v?.id) {
//            R.id.button -> {
////                val editTextStr = editText.text.toString()
////                Toast.makeText(this, editTextStr, Toast.LENGTH_SHORT).show()
////                imageView.setImageResource(R.drawable.img_2)
////                if (progressBar.visibility == View.VISIBLE) {
////                    progressBar.visibility = View.GONE
////                } else {
////                    progressBar.visibility = View.VISIBLE
////                }
//
//                AlertDialog.Builder(this).apply {
//                    setTitle("This is Dialog")
//                    setMessage("Something is Dialog")
//                    setCancelable(false)
//                    setPositiveButton("OK"){ dialog, which ->
//                        progressBar.progress = progressBar.progress + 10
//                    }
//                    setNegativeButton("Cancel"){ dialog, which ->
//
//                    }
//                    show()
//                }
//
//            }
        }
    }
}